doc/lua_api.txt

```diff
@@ -7223,7 +7223,7 @@ Used by `minetest.register_node`, `minetest.register_craftitem`, and
         -- The user may be any ObjectRef or nil.
         -- The default functions handle regular use cases.
 
-        after_use = function(itemstack, user, node, digparams),
+        after_use = function(itemstack, user, node, digparams, pointed_thing),
```

Comment:
I suggest stating here that `pointed_thing` can be `nil` (and can be pointing to nothing, maybe).